cmd/flow-dps-indexer/main.go

```diff
@@ -90,6 +92,13 @@ func run() int {
 	}
 	defer data.Close()
 
+	_, err = index.NewReader(db).Last()
```

Comment:
I think we should check the `First()` call, that's kind of the stable thing that will never change once we start indexing a spork.